src/actions/types.js

```diff
@@ -7,8 +7,17 @@ export const SOCIAL_LOGIN_SUCCESS = 'SOCIAL_LOGIN_SUCCESS';
 export const GET_PROFILE_PAYLOAD = 'GET_PROFILE_PAYLOAD';
 export const GET_PROFILE_INITIATED = 'GET_PROFILE_INITIATED';
 
-export const GET_ALL_ARTICLES_SUCCESS = 'GET_ALL_ARTICLES_SUCCESS';
 export const CREATE_ARTICLE_SUCCESS = 'CREATE_ARTICLE_SUCCESS';
 export const CREATE_ARTICLE_INITIATED = 'CREATE_ARTICLE_INITIATED';
 export const CREATE_ARTICLE_ERROR = 'CREATE_ARTICLE_ERROR';
 export const LOGOUT_USER = 'LOGOUT_USER';
+export const SEND_RESET_LINK_SUCCESS = 'SEND_RESET_LINK_SUCCESS';
```

Comment:
Please remove these reset password changes

Reply:
@RonKbS Reset-password changes have been removed after rebasing develop branch

Reply:
@samuelbwambale these particular action_types should also be removed since the updated PR for them has not yet been merged.

Reply:
@RonKbS The action_types have been removed